src/createServer.js

```diff
@@ -1,3 +1,67 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const [pathName, queryParams] = req.url.split('?');
+    const text = pathName.slice(1);
+    const params = new URLSearchParams(queryParams);
+    const caseParams = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+    const toCase = params.get('toCase');
+
+    const errors = [];
+
+    if (text.length === 0) {
+      errors.push({
+        // eslint-disable-next-line max-len
+        message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
create object where you would store your error messages, also create another file for this